src/components/ProductCard.tsx

```diff
@@ -27,11 +29,13 @@ export const ProductCard = ({
   const { t } = useTranslation()
   const [open, setOpen] = useState(false)
   const dispatch = useDispatch()
-  const orderName = useSelector(
-    (state: RootState) =>
-      state.order.value.find((order) => order.products.find((id) => id))
-        ?.title ?? 'none',
+  const orderNameSelector = createSelector(
```

Comment:
All selectors should be in separate module

Reply:
fixed